instruments/technical_docs_for_high_risk_ai.yaml

```diff
@@ -0,0 +1,378 @@
+systemcard_path: .assessments[]
+schema_version: 1.1.0
+
+instrument:
+  name: "Technische documentatie voor hoog-risico AI-systemen"
+  description: "De technische documentatie van een AI-systeem met een hoog risico wordt opgesteld voordat dit systeem in de handel wordt gebracht of in gebruik wordt gesteld."
+  urn: "urn:nl:aivt:ir:td:1.0"
+  language: "nl"
+  owners:
+    - organization: "European Union"
+    - email: ""
+    - role: ""
+    - date: ""
+    - url: "https://www.europarl.europa.eu/doceo/document/TA-9-2024-0138-FNL-COR01_NL.pdf" #See article 11 and annex IV of European AI Act
+
+template: &assessmentcard_template                  # anchor for all fields belonging to answer
+  question: "$QUESTION"
+  answer: "$ANSWER"
+  remarks: "$REMARKS"
+  timestamp: "$TIMESTAMP"
+  authors:
+    - name: "$AUTHOR.NAME"
+      email: "$AUTHOR.EMAIL"
+      role: "$AUTHOR.ROLE"
+
+# Algemene beschrijving AI-systeem
+tasks:
+  - question: "Wat is het beoogde doel van het AI-systeem?"
+    urn: "urn:nl:aivt:ir:td:1.0:1.a.1"
+    labels:
+      - key: topic
+        value: "Algemene beschrijving AI-systeem"
+    type: "open_question"
+    suggested_roles: [ ]
+    lifecycle: [ ]
+    links: [ {"text": "DPIA vraag 4"}, {"urn": "urn:nl:aivt:ir:1.0:1.2.1"}, {"text": "system_card0.1a4:  goal_and_impact"}]     #TO DO: find a way to link to specific system card fields
```

Comment:
What do we mean with `{"text": "system_card0.1a4:  goal_and_impact"}`?

I would have expected this something like:

```json
{"systemcard_path": ". goal_and_impact", "version": ">0.1a1"}
```

Reply:
Is the definition of the 'links' field already defined somewhere? (I could not find it in the schema definitions https://minbzk.github.io/ai-validation/projects/tad/reporting-standard/latest/#system_card). I assume we would want that. I also wonder what 'text: DPIA vraag 4' refers to.. is it a local reference within the same file.. ?

Reply:
We were looking for a way to accommodate for using the 'links' field to link between different files (other instruments and/or system cards). 

The definition of the 'links' can be found in the schema.json in this repo: "links" -
            "description": "Links to other relevant measures / tasks". 

Furthermore the link to the DPIA ('DPIA vraag 4') refers to an instrument that we have not yet build. Therefore it does not have an urn yet.

Reply:
@uittenbroekrobbert the schema for instruments can be found in THIS repository (it's not part of the reporting standard)